src/serlo-editor/plugins/audio/editor.tsx

```diff
@@ -27,21 +30,35 @@ export const AudioEditor = (props: AudioProps) => {
         </div>
       ) : (
         <div className="pb-8 pt-8">
-          <>
-            {focused && (
-              <div className="flex justify-around space-x-4">
-                <div
-                  className="flex cursor-pointer flex-col items-center"
-                  onClick={() => setShowSettingsModal(true)}
-                >
-                  <FaIcon icon={faLink} className="text-2xl" />
-                  <span className="mt-2">Enter audio URL</span>
-                </div>
-              </div>
-            )}
-          </>
+          <ShowAudioSettingsButton
+            openSettings={() => setShowSettingsModal(true)}
+          >
+            {audioUrl}
+          </ShowAudioSettingsButton>
         </div>
       )}
     </>
   )
 }
+
+interface ShowAudioSettingsButtonProps {
+  openSettings: () => void
+  children: JSX.Element | string
+}
+
+export function ShowAudioSettingsButton({
+  openSettings,
+  children,
+}: ShowAudioSettingsButtonProps) {
+  return (
+    <div className="flex justify-around space-x-4">
+      <button
+        className="flex cursor-pointer flex-col items-center border-none bg-transparent p-0"
+        onClick={openSettings}
+      >
+        <FaIcon icon={faLink} className="text-2xl" />
+        <span className="mt-2">{children}</span>
+      </button>
+    </div>
+  )
+}
```

Comment:
Sorry, I know this is a cleaner pattern than the `render…` stuff, but if you abstract it as a component I'd prefer if you would put it into an own file.